js/main.js

```diff
@@ -251,6 +276,81 @@ function directorInit() {
   });
 
 }
+// OVERLAY IMAGE
+var overlayImage = {
+
+  player: $('#image-overlay-player'),
+
+  overlay: $('#image-overlay'),
+  overlayInner: $('#image-overlay-inner'),
+
+  timeout: 0,
+
+  load: function(image, postIndex) { 
+    var _this = this;
+
+    image.attr('width', '100%').removeAttr('height');
+
+    _this.overlay.fadeIn(fastAnimationSpeed).data('now-playing', postIndex);
+    _this.player.html(image);
+
+    // use fixHeight after animations
+      _this.fixHeight();
+    
+    $(window).resize(function() {
```

Comment:
probably not needed anymore as there is no longer an opening animation from the player